pkgs/development/libraries/libcbor/default.nix

```diff
@@ -13,28 +13,28 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "libcbor";
-  version = "0.10.0";
+  version = "unstable-2023-01-29";
```

Comment:
Can you add a short note why latest release is not good enough? Would the next one be good enough?

Reply:
Done